cmd/antrea-agent/agent.go

```diff
@@ -353,31 +360,35 @@ func run(o *Options) error {
 		}
 	}
 
-	isChaining := false
-	if networkConfig.TrafficEncapMode.IsNetworkPolicyOnly() {
-		isChaining = true
-	}
-	cniServer := cniserver.New(
-		o.config.CNISocket,
-		o.config.HostProcPathPrefix,
-		nodeConfig,
-		k8sClient,
-		isChaining,
-		enableBridgingMode, // activate AntreaIPAM in CNIServer when bridging mode is enabled
-		routeClient,
-		networkReadyCh)
-
+	isWorkerNode := agentRole == types.CNIAgent
```

Comment:
Why use all these different names (isWorkerNode, CNIAgent, Node)? Let us just use KubernetesNode, and maybe K8sNode for short.

Reply:
updated.